### ISSUE TITLE
Allow nested multiline comments

### ISSUE THREAD

**Issue by robertjlooby** [avh4/elm-format#110]
The [Elm syntax reference](http://elm-lang.org/docs/syntax) gives this example of multiline comment syntax:

```
{- a multiline comment
   {- can be nested -}
-}
```

Formatting this removes the nested comment leaving:

```
{- a multiline comment

-}
```

**Comment by rtfeldman** [comment]
I guess the use case for this is when you want documentation which includes a code example which includes a comment? Yowza.

**Comment by avh4** [comment]
Fixed in ab444cc.
